src/tests/BundleApi.test.ts

```diff
@@ -135,6 +135,34 @@ export class BundleApiTest {
         should(bundle2.context.plugins).deepEqual(["bar"]);
     }
 
+    "Should set extension overrides"() {
+        const fuse = createFuse();
+        const bundle = fuse.bundle("app")
+            .extensionOverrides(".foo.js", ".foo.json")
+            .extensionOverrides(".foo.css", ".foo.less");
+
+        should(bundle.context.extensionOverrides.overrides).deepEqual([".foo.js", ".foo.json", ".foo.css", ".foo.less"]);
+    }
+
+    "Should not add an extension override if it is invalid"() {
+        const fuse = createFuse();
+        const bundle = fuse.bundle("app")
+            .extensionOverrides("foo.js", ".foo.json");
+
+        should(bundle.context.extensionOverrides.overrides).deepEqual([".foo.json"]);
+    }
+
+    "Should not share extension overrides across bundles"() {
+        const fuse = createFuse();
+        const bundle1 = fuse.bundle("app")
+            .extensionOverrides(".foo.js", ".foo.json");
+        const bundle2 = fuse.bundle("app")
+            .extensionOverrides(".bar.js", ".bar.json");
+
+        should(bundle1.context.extensionOverrides.overrides).deepEqual([".foo.js", ".foo.json"]);
+        should(bundle2.context.extensionOverrides.overrides).deepEqual([".bar.js", ".bar.json"]);
+    }
```

Comment:
Not enough tests, we should make similar tests additionally

https://github.com/fuse-box/fuse-box/blob/8586d90baac5326f849aadf8aca82377a7e33764/src/tests/plugins/CSSPlugin.test.ts#L131-L147

To cover all the cases, not just VuePlugin

Reply:
Im not sure I understand? The tests are in the bundle? I should probably add some for `File` though for sure.

Reply:
No, I just gave you an example. You can create test bundle in a test, with the actual file structure and stuff.